configs/experiment/biofoundation/linear_classifier/audiomae_HSN.yaml

```diff
@@ -0,0 +1,67 @@
+# @package _global_
```

Comment:
@yokoryu-meteor you based this branch on your audioMAE branch and not main, thats why you also have included the audiomae changes. Please remove them from this branch (rebase on main / revert the commits)

Reply:
I remvoed those changes.